prow/git/v2/adapter.go

```diff
@@ -91,7 +91,8 @@ func (a *repoClientAdapter) MirrorClone() error {
 }
 
 func (a *repoClientAdapter) Fetch(arg ...string) error {
-	return a.Repo.Fetch(arg...)
+	args := append([]string{"origin"}, arg...)
```

Comment:
why not make this part of the call?

Reply:
This is a bug with the gitv1 -> v2 adapter. In the v2 client Fetch does `git fetch remote args...` but in v1 it does `git fetch args...` This change brings the v1 client in line with the v2 client I believe

Reply:
Added a todo to figure out how to fix the differing functionality without hard-coding origin